data/human/sales.txt

```diff
@@ -349,6 +351,7 @@ outfitter "Deep Sky Advanced"
 	"In-flight Mass Display"
 	"Hyperjump Fuel Display"
 	"Fuel Bar Hyperjump Display"
+	"Astronomical Scanner"
```

Comment:
These are a good place for this outfit for now, although in the future it might be more interesting to have it in its own outfitter and have it only available from a few worlds that mention having universities or astronomy studies, and tie it into some research missions from said places.

Reply:
I have it in Deep Sky Basics and Deep Sky Advanced currently, is just Advanced preferred?

Reply:
Probably just advanced, I think.